qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -408,9 +459,10 @@ def config(self, clock_source=None, sample_rate=None, clock_edge=None,
                            self.parameters['coupling' + str(i)],
```

Comment:
handle should be binary, i.e. `2**(i-1)`. This should only matter for 4-channel ATS, like the ATS9440